.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+name: CI
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  job1:
+    name: Nim Tests
+    runs-on: ubuntu-18.04
+    container: nimlang/nim:1.2.4-ubuntu-regular@sha256:02a555518a05c354ccb2627940f6138fca1090d198e5a0eb60936c03a4875c69
+
+    steps:
+    - uses: actions/checkout@aabbfeb2ce60b5bd82389903509092c4648a9713 # v2.2.0
+
+    - name: Compile and run `tests/*`
```

```suggestion
    - name: Compile and run `tests/t*.nim` files
```

Or something similar. I'm also fine with: 
- "Run \`nimble test\`"
- "Compile and execute tests"

It's just that, strictly speaking, `tests/*` is wrong. As I'm sure you know, `nimble test` only operates on files (or links) that:
- are in the root of the `tests` directory
- start with `t`
- end with `.nim`

The relevant `nimble test` code is [here](https://github.com/nim-lang/nimble/blob/45ad606387f9c12d3d5f27957bcf5def9c0cd5e5/src/nimble.nim#L1111-L1113).